# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../kiwi/hotfix_hyperlink.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

http://bugs.async.com.br/attachment.cgi?id=3657

Index: ./kiwi/ui/hyperlink.py
===================================================================
--- ./kiwi/ui/hyperlink.py	(revision 4967)
+++ ./kiwi/ui/hyperlink.py	(working copy)
@@ -68,6 +68,8 @@
         @type text: str
         """
         gtk.EventBox.__init__(self)
+        self.set_above_child(False)
+        self.set_visible_window(False)
         PropertyObject.__init__(self)
         self._gproperties = {}
         if text is not None:
